recipes-devtools/maccalc/maccalc.bb

```
@@ -0,0 +1,21 @@
+#
+# This file was derived from the 'Hello World!' example recipe in the
+# Yocto Project Development Manual.
+#
+SUMMARY = "MAC address manupulation utility"
```

Typo: manipulation. OTOH. it's direct from the author's mouth.
```suggestion
SUMMARY = "MAC address manipulation utility"
```

---

C&P without noticing... will fix anyway